docs/tutorials/PickupGuide.md

```diff
@@ -0,0 +1,77 @@
+PickupGuide
+===========
+
+### From open.mp Wiki
+
+  
+
+
+### Define the pickupid
+
+The first thing to be done when creating pickups is creating a place to store their ID. This will be done in a global variable so it can be set when you create the pickup and read when you pick up a pickup, calling a callback with the ID of the pickup you picked up. For this example we will use the name "mypickup".
+
+new mypickup;
+
+
+### Creating the pickup
+
+There are two ways to create pickups. [CreatePickup](../functions/CreatePickup "CreatePickup") and [AddStaticPickup](../functions/AddStaticPickup "AddStaticPickup"). AddStaticPickup doesn't return an ID when it is created, can't be destroyed and can only be used under OnGameModeInit, so for this example we will use [CreatePickup](../functions/CreatePickup "CreatePickup").
```

Comment:
Markdown hyperlinks don't need the additional text at the end.

Also these links are 404s because there's no `scripting/` path component.